src/test/java/bio/terra/service/filedata/azure/tables/LoadHistoryStorageTableConnectedTest.java

```diff
@@ -75,38 +79,86 @@ public void setup() throws Exception {
 
   @After
   public void cleanup() throws Exception {
-    // TODO - clean up the load history table
+    storageTableDao.deleteLoadHistory(datasetId, serviceClient);
     connectedOperations.teardown();
   }
 
   @Test
   public void testStorageTableMetadataDuringFileIngest() {
-    String loadTag = UUID.randomUUID().toString();
     List<BulkLoadHistoryModel> loadHistoryArray = new ArrayList<>();
+    // ---- TEST FOUR STATES OF LOAD HISTORY TABLE ENTRIES ----
+    // SUCCEEDED
+    String checksum = Names.randomizeName("CRCSUM");
+    String successfulSourcePath = "/" + UUID.randomUUID().toString() + "/source/path.json";
```

Comment:
You don't need `.toString()` if an object is part of a string concatenation expression. The IDE can highlight these cases and fix them for you automatically.
```suggestion
    String successfulSourcePath = "/" + UUID.randomUUID() + "/source/path.json";
```